src/DAT/DAT.jl

```diff
@@ -577,7 +577,7 @@ function generateOutCube(::Type{T},eltype,oc::OutputCube,loopcachesize,co;kwargs
       cs = Base.setindex(cs,cc,i)
     end
   end
-  cube1, cube2 = createdataset(T, oc.allAxes; chunksize=cs, chunkoffset=co, kwargs...)
+  cube1, cube2 = createdataset(T, oc.allAxes; T = eltype, chunksize=cs, chunkoffset=co, kwargs...)
```

Comment:
The eltype variable here shadows Base.eltype, right? 
Should we rename this variable to maybe `elementtype`?
This doesn't have to happen in this pull request, I could prepare a PR to rename it consistently.

Reply:
Thanks I just fixed this